dotnet/samples/KernelSyntaxExamples/Program.cs

```diff
@@ -21,7 +21,7 @@ public static async Task Main(string[] args)
         using CancellationTokenSource cancellationTokenSource = new();
         CancellationToken cancelToken = cancellationTokenSource.ConsoleCancellationToken();
 
-        string? defaultFilter = null; // Modify to filter examples
+        string? defaultFilter = "Example65"; // Modify to filter examples
```

Comment:
Move back the the original = null,

If using VS change that in the launchSettings 😊

![image](https://github.com/microsoft/semantic-kernel/assets/19890735/66a28391-d3a1-44de-a5c0-79fc47ba656f)

![image](https://github.com/microsoft/semantic-kernel/assets/19890735/cb84f3eb-f506-4085-a787-85323e4330e8)